jobs/validate/nvidia-spec

```diff
@@ -14,18 +14,23 @@ set -x
 ###############################################################################
 # FUNCTION OVERRIDES
 ###############################################################################
+
+# Reference: https://ubuntu.com/kubernetes/docs/gpu-workers
 function juju::deploy
 {
     tee overlay.yaml <<EOF > /dev/null
-series: $SERIES
+SERIES: $SERIES
 applications:
+  containerd:
+    options:
+      gpu_driver: none
```

Comment:
OK -- so containerd will NOT be installing the nvidia drivers?  Somehow this gets delegated to the nvidia-gpu-operator charm?

Reply:
Indeed!